src/Http/Client.php

```diff
@@ -479,7 +488,9 @@ public function send(RequestInterface $request, array $options = []): Response
         }
 
         do {
+            $this->dispatchEvent('Client.beforeSend', [$request, $options]);
             $response = $this->_sendRequest($request, $options);
+            $this->dispatchEvent('Client.afterSend', [$request, $options, $response]);
```

Comment:
```suggestion
            $this->dispatchEvent('Client.afterSend', [$request, $response, $options]);
```

Generally we always have $options as the last parameter even when methods have different numbers of parameters. `beforeMarshall` and `afterMarshall` are an example of that.